web-js/4.x/download-csv/README.md

```diff
@@ -0,0 +1,4 @@
+## Download Client Side CSV
+I have seen a lot of questions about how to save data from an ArcGIS JavaScript application. There are a few ways built in, such as applyEdits and using a Geoprocessing service. Both of those options are server side operations, so I thought adding a client side option may help some people. In this sample the user can add points to a map. When they click the download button, the points along with queried feature layer attributes are downloaded as a CSV.
+
+[Live Demo on Codepen](https://codepen.io/benesri/pen/ZEzyQNm)
```

Comment:
Instead of using CodePen, could you use the format of: https://esri.github.io/developer-support/web-js/4.x/download-csv/index.html so that it's using the actual code hosted on GitHub?  

If CodePen is used, your account could expire and the link could then be broken. Thanks!